vendor/vendor.json

```diff
@@ -619,5 +634,5 @@
 			"revisionTime": "2017-09-19T05:20:08Z"
 		}
 	],
-	"rootPath": "github.com/trackit/trackit-server"
+	"rootPath": "github.com/trackit/trackit"
```

Comment:
If this is changed only in this PR, does it mean that if we deploy stg right now it will be broken ?

Reply:
Nevermind, this fix was already merged in `stg`